umbraco/Umbraco-CMS
# Fix for issue #5816

### Prerequisites
PR for the fix for Issue #5816 

**Steps to verify the fix**
Navigate to Umbraco > Settings >Document Type > Choose a document type
Select Templates from the top right corner

Verify the text under the template name and  extra template

All karma unit tests passed 

### Description
To add a space between the translated text and the text that is injected via binding a &nbsp; was added in between the text in the markup.
Please find below the screenshot after fix


![Fix 5816](https://user-images.githubusercontent.com/1662197/60971160-e118c780-a366-11e9-911e-e7e44dccb4dc.PNG)
